okonomiyaki/file_formats/setuptools_egg.py

```diff
@@ -74,37 +79,84 @@ def _get_default_setuptools_abi(platform_string, pyver):
             raise ValueError(msg)
 
 
+class _UnspecifiedArg(object):
+    pass
+_UNSPECIFIED = _UnspecifiedArg()
```

Comment:
Hm, for some reason I thought you could not create instances of object directly !